Lib/http/server.py

```diff
@@ -105,6 +106,12 @@
 import time
 import urllib.parse
 
+try:
+    import ssl
```

Comment:
Don't try this at the top level as many users don't need it. Do it in the HTTPSServer constructor.